### PR TITLE
Add instructions to create a directory before running `rustup override`

### LINKED ISSUE #157: install instructions mention a directory before telling you to create one
specifically it says to run `rustup override add nightly`, says something about how it'll make this directory use nightly, but it hasn't said to make a directory or which one to be in yet. this could lead to people using nightly rust everywhere on accident, which might make them pretty sad.

### PR DESCRIPTION
**Pull request by jdanford**
Fix #157

### CODE REVIEW

File: src/installing-rust.md [hunk 1]
```diff
@@ -15,18 +15,25 @@ $ rustup update nightly
...
+in this book under continuous integration, so we should know if something
 changes in nightly Rust and breaks. But please [file bugs] if something doesn't
 work.
 
 [file bugs]: https://github.com/intermezzOS/book/issues/new
 
-Then, execute this:
+You shouldn't use nightly Rust unless you need to, which is why `rustup` allows
+you to override the default version whenever you're in a particular directory.
+We don't have a project directory yet, so let's create one:
```
Comment (steveklabnik) [inline src/installing-rust.md]: So, we do this over at http://intermezzos.github.io/book/multiboot-headers.html#writing-our-own-multiboot-header

I can see why doing it here is also nicer, but we should remove that as well.

**Review by steveklabnik — CHANGES_REQUESTED** [review]

**Comment by jdanford** [comment]
My bad, I figured it had to exist somewhere but I didn't look hard enough! Fixed.

File: src/installing-rust.md (hunk 1 above)
Comment (jdanford) [reply src/installing-rust.md]: This has been fixed, not sure if you saw the other comment! Hopefully force pushing wasn't the wrong way to do it.

Comment (steveklabnik) [reply src/installing-rust.md]: It's not, it's the perfect way. Thanks a ton!

**Review by steveklabnik — APPROVED** [review]

**Pull request merged**